feat: Added Relationships page

soluchok opened the pull request:
<img width="1760" alt="Screen Shot 2020-08-05 at 9 20 15 PM" src="https://user-images.githubusercontent.com/12339578/89449304-8d430280-d761-11ea-974a-427b6909bd07.png">
<img width="1780" alt="Screen Shot 2020-08-05 at 9 19 56 PM" src="https://user-images.githubusercontent.com/12339578/89449308-8e742f80-d761-11ea-8166-ebace69ca406.png">
<img width="1785" alt="Screen Shot 2020-08-05 at 9 19 45 PM" src="https://user-images.githubusercontent.com/12339578/89449310-8f0cc600-d761-11ea-92ce-ac19f615f9aa.png">
<img width="1769" alt="Screen Shot 2020-08-05 at 9 19 21 PM" src="https://user-images.githubusercontent.com/12339578/89449313-8fa55c80-d761-11ea-97f1-daa01a239960.png">

Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

llorllale commented:
@soluchok I'm assuming this PR is just setting it up for followups that "make it pretty" right?

soluchok commented:
> @soluchok I'm assuming this PR is just setting it up for followups that "make it pretty" right?

Yes. Also, I removed some clicks.
This PR includes:
* generates invitation (by default)
* button to copy invitation (no need to select it)
* split connections (now we have pending and completed)
pending requires one more click
* added badge when we have pending requests

Currently, it requires a router set up. I will try to register the router automatically.
